survey_partner_representative/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2024 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Survey Partner Representative",
+    "summary": "Fill the survey on behalf of others",
+    "version": "15.0.1.0.0",
+    "development_status": "Beta",
+    "category": "Marketing/Survey",
+    "website": "https://github.com/OCA/survey",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "maintainers": ["chienandalu"],
+    "license": "AGPL-3",
+    "depends": ["survey"],
+    "data": [
+        "security/survey_partner_representative_security.xml",
+        "views/survey_survey_views.xml",
+        "views/survey_user_input_views.xml",
+    ],
+    "demo": ["demo/survey_partner_representative_demo.xml"],
+    "external_dependencies": {
+        "python": ["odoo_test_helper"],
```

Comment:
Don't add it here. It's only for tests. Instead, create a file `test_requirements.txt` for being installed on runboat and CI.